assets/konnectivity/konnectivity-agent-data-plane-daemonset.yaml

```diff
@@ -66,6 +66,7 @@ spec:
           name: konnectivity-ca
         - mountPath: /etc/konnectivity/agent
           name: agent-certs
+      dnsPolicy: Default
```

Comment:
You need to `make bindata` for this change.

Reply:
Yep, sorry. Done!